apps/full-site-editing/full-site-editing-plugin/readme.txt

```diff
@@ -40,6 +40,11 @@ This plugin is experimental, so we don't provide any support for it outside of w
 
 == Changelog ==
 
+= 1.10 =
+* Turned off site editor demo templates
```

Comment:
I checked my local version of the branch I stil had lying around -- I have

```suggestion
* Improvements to the premium blocks.
```

here. The rest is 100% identical :slightly_smiling_face:

Reply:
Thanks Bernie